tests/testthat/test_inspectDifferences.R

```diff
@@ -4,34 +4,34 @@ load(file = "helper_data.rda")
 
 test_that("Errors",{
   df1_5 <- df1_4 <- df1_2 <- df1_3 <- df1
-  expect_error(inspectDifferences(c("1", "2"), df1, df1_2, id = "ID1"),
-               "'varName' must be a character of length 1.")
-  expect_error(inspectDifferences("V1", df1, df1_2, id = 1),
-               "'id' must be a character of length 1.")
-  expect_error(inspectDifferences("V2", df1, df1_2, id = "ID1"),
-               "'varName' is not a variable in 'GADSdat1'.")
+  expect_error(inspectDifferences(df1, varName = c("1", "2"), id = "ID1"),
+               "'varName' needs to be a character vector of length 1.")
+  expect_error(inspectDifferences(df1, varName = "V1", id = 1),
+               "'id' needs to be a character vector of length 1.")
+  expect_error(inspectDifferences(df1, varName = "V2", id = "ID1"),
+               "The following 'varName' are not variables in the GADSdat: V2")
   df1_2$dat <- df1_2$dat[c(1, 1, 2), ]
-  expect_error(inspectDifferences("V1", df1, df1_2, id = "ID1"),
-               "'GADSdat1' and 'GADSdat2' have different row numbers.")
+  expect_error(inspectDifferences(df1, varName = "V1", other_GADSdat = df1_2, id = "ID1"),
+               "'GADSdat' and 'other_GADSdat' have different row numbers.")
   df1_3$dat$ID1 <- c(2, 1)
-  expect_error(inspectDifferences("V1", df1, df1_3, id = "ID1"),
-               "'id' column is not equal for 'GADSdat1' and 'GADSdat2'.")
+  expect_error(inspectDifferences(df1, varName = "V1", other_GADSdat = df1_3, id = "ID1"),
+               "'id' column is not equal for 'GADSdat' and 'other_GADSdat'.")
   df1_4$dat[1, 1] <- NA
-  expect_error(inspectDifferences("V1", df1, df1_4, id = "ID1"),
-               "Missing values in 'id' column of 'GADSdat2'.")
-  expect_error(inspectDifferences("V1", df1_4, df1, id = "ID1"),
-               "Missing values in 'id' column of 'GADSdat1'.")
+  expect_error(inspectDifferences(df1, varName = "V1", other_GADSdat = df1_4, id = "ID1"),
+               "Missing values in 'id' column of 'other_GADSdat'.")
+  expect_error(inspectDifferences(df1_4, varName = "V1", id = "ID1"),
+               "Missing values in 'id' column of 'GADSdat'.")
   df1_5$dat[, 2] <- as.character(df1_5$dat[, 1])
-  expect_error(inspectDifferences("V1", df1, df1_5, id = "ID1"),
-               "'varName' column is numeric in 'GADSdat1' but not in 'GADSdat2'.")
-  expect_error(inspectDifferences("V1", df1_5, df1, id = "ID1"),
-               "'varName' column is numeric in 'GADSdat2' but not in 'GADSdat1'.")
+  expect_error(inspectDifferences(df1, varName = "V1", other_GADSdat = df1_5, id = "ID1"),
+               "'varName' column is numeric in 'GADSdat' but not in 'other_GADSdat'.")
+  expect_error(inspectDifferences(df1_5, varName = "V1", other_GADSdat = df1, id = "ID1"),
+               "'varName' column is numeric in 'other_GADSdat' but not in 'GADSdat'.")

```

Comment:
Please add to the "expect-error"-section a test where argument "other_varName" is used